R/data-converters.R

```diff
@@ -65,19 +65,33 @@ enw_add_cumulative <- function(obs, by = NULL, copy = TRUE) {
 #' # Make use of maximum reported to calculate empirical daily reporting
 #' dt <- enw_add_max_reported(dt)
 #' enw_add_incidence(dt)
+
+# --------------
+##  temporary placeholder
+enw_check_date_sequence  <- function(obs, by=NULL, copy=TRUE) {
```

Comment:
This looks good I think but is throwing a few linting issues that it would be good to address and would ideally follow our style guide (i.e explicit returns etc)